ui/src/app/shared/status/status.icon.html

```diff
@@ -1,9 +1,9 @@
 <div class="status-icon" [ngSwitch]="status">
     <div class="wrapper">
         <ng-container *ngSwitchCase="pipelineStatusEnum.BUILDING">
-            <div *ngIf="loader" class="ui small active inline blue loader">
-                <span class="">{{value}}</span>
-            </div>
+            <i *ngIf="loader">
+                <img id="gifloader" src="/assets/images/loader.gif">
```

Comment:
what about <span class="">{{value}}</span> ?
If not use anymore, clean entire component